inductiva/_cli/cmd_storage/remove.py

```diff
@@ -1,20 +1,46 @@
 """Remove the user's remote storage contents via CLI."""
 
-from inductiva import storage
+import sys
+
+from inductiva import constants, storage
 
 
 def remove(args):
     """Remove user's remote storage contents."""
-    path = args.path
+    paths = args.path
     confirm = args.confirm
+    all_paths = args.all
+
+    if paths and all_paths:
+        print(
+            "inductiva storage remove: error: "
+            "argument path not allowed with argument --all",
+            file=sys.stderr)
+        sys.exit(1)
 
     if not confirm:
-        prompt = input(f"Are you sure you want to remove {path}? (y/n)")
+        if all_paths:
+            prompt = input("You are about to remove EVERYTHING from your"
+                           " remote storage space.\n"
+                           "Are you sure you want to proceed (y/[N])? ")
+        else:
+            if len(paths) > constants.MAX_CONFIRMATION_LINES:
+                prompt = input(f"You are about to remove {len(paths)} "
+                               "paths from your remote storage space.\n"
+                               "Are you sure you want to proceed (y/[N])? ")
+            else:
+                print("You are about to remove the following paths from "
+                      "your remote storage space: ")
+                for path in paths:
+                    print(f"  - {path}")
+                prompt = input("Are you sure you want to proceed (y/[N])? ")
```

Comment:
Just a single prompt:

```suggestion
            prompt = input("Are you sure you want to proceed (y/[N])? ")
```